### ISSUE TITLE
Implement legal notification of viable meetups

### ISSUE THREAD

**Issue by c3o** [hack4vdb/anthill-backend#54]
Müssen wir die Treffen nun als Versammlungen anmelden oder nicht??

**Comment by anonjum** [comment]
Ja!

Jeweils bei der zuständigen Landespolizeidirektion bis 72 Stunden vor der Veranstaltung. Wenn kein Widerspruch rein kommt gilt die Veranstaltung als genehmigt. Anbei eine Muster Nachricht. Die Anmeldung kann formlos via E-Mail erfolgen.

Es müsste hier ein Kundgebungsleiter genannt werden. Das muss die Person sein, die die Flyer erhält. Diese Information müssen wir der Person vermitteln und sie muss uns erlauben die Daten an die Polizei weiter zu leiten. Aber wenn wir die Telefonzentrale haben bin ich zuversichtlich, dass wir da Menschen gewinnen können.

![bildschirmfoto 2016-08-21 um 15 57 46](https://cloud.githubusercontent.com/assets/21046499/17837627/1be87f4c-67b8-11e6-8a6f-65d9b5f60bd9.png)

Ich bin nicht sicher, ob die Zusendung an diese E-Mailadressen passt, oder ob es eigene sind. Die von NÖ steht in dem Bild drinnen, die anderen müsste man sich holen.

Landespolizeidirektionen
LPD Burgenland
Neusiedler Straße 84
7000 Eisenstadt
Telefon: 059133/10/0
Fax: 059133/10/1009
LPD-B@polizei.gv.at
LPD Kärnten
Buchengasse 3
9020 Klagenfurt am Wörthersee
Telefon: 059133/20/0
Fax: 059133/20/1009
LPD-K@polizei.gv.at
LPD Niederösterreich
Neue Herrengasse 15
3100 St. Pölten
Telefon: 059133/30/0
Fax: 059133/30/1009
LPD-N@polizei.gv.at
LPD Oberösterreich
Gruberstraße 35
4021 Linz
Telefon: 059133/40/0
Fax: 059133/40/7803
LPD-O@polizei.gv.at
LPD Salzburg
Alpenstraße 90
5020 Salzburg
Telefon: 059133/50/0
Fax: 059133/50/7800
LPD-S@polizei.gv.at
LPD Steiermark
Straßganger Straße 280
8052 Graz
Telefon: 059133/60/0
Fax: 059133/60/1009
LPD-ST@polizei.gv.at
Parkring 4 (Ämtergebäude)
8010 Graz
Telefon: 059133/60-0
LPD-ST@polizei.gv.at
LPD Tirol
Standort
Kaiserjägerstraße 8
6020 Innsbruck
Telefon: 059133/70/0
Fax: 059133/70/7888
Standort
Innrain 34
6020 Innsbruck
Telefon: 059133/70/0
Fax: 059133/70/7800
LPD-T@polizei.gv.at
LPD Vorarlberg
Bahnhofstraße 45
6900 Bregenz
Telefon: 059133/80/0
Fax: 059133/80/1009
LPD-V@polizei.gv.at
LPD Wien
Schottenring 7-9
1010 Wien
Telefon: 01 31310-0
LPD-W@polizei.gv.at

**Comment by c3o** [comment]
Uff 😓

**Comment by anonjum** [comment]
Ja, ist not so hot. Aber das sind die Gesetze und prinzipiell ist das ja auch sinnvoll. Ich finde es auch wieder gut, weil es dann echtes commitment von der Person ist. Und dann finden vll weniger veranstaltungen statt, aber die die stattfinden, da fühlt sich jemand verantwortlich.

Dementsprechend müssen wir, denke ich die automatisch generierten Events ausblenden oder zumindest ganz klar als Vorschläge kennzeichnen!

Wir brauchen die Möglichkeit selbst ein Event anzulegen mit eigener Uhrzeit und eigenem Datum. In dem Form muss drinnen stehen: 
1. Danke!
2. Du bist jetzt verantwortlich
3. Wir schicken Dir Flyer zu
4. Wir registrieren die Veranstaltung auf Deinen Namen

und als Alternative: Ich nehme lieber an einer Veranstaltung teil, für die jemand anders verantwortlich ist. Es ist härter, und es ist unsexier. Aber es ist ehrlich. Und wir werden Leute haben, die telefonieren werden - das heißt wir bekommen die Leute motiviert. Ich bin zuversichtlich.

Und bitte: Wir müssen bei allen die Handynummer abfragen (auch bei den einfachen Teilnehmern) und es muss dabei stehen: das Team Van der Bellen darf mich elektronisch und telefonisch kontaktieren.

**Comment by anonjum** [comment]
Ich möchte die Bundespräsidentschafts-Kampagne von Alexander Van der Bellen unterstützen! Ich erkläre mich damit einverstanden, dass mich der Verein "Gemeinsam für Van der Bellen" diesbezüglich elektronisch und telefonisch kontaktiert, mir die bestellten Werbemittel zusendet, sowie mich über den weiteren Verlauf der Kampagne informiert.
